lambda/index.ts

```diff
@@ -17,9 +17,10 @@ const STATE_MESSAGES: { [k in keyof typeof WorkflowState]: [color: string, getSt
   finished:                [COLORS.success, ref => `*Finished* deploying ${ref}`],
   stopped:                 [COLORS.warning, ref => `Deployment *stopped* for ${ref}`],
   failed:                  [COLORS.error,   ref => `Deployment *failed* for ${ref}`],
+  skipped:                 [COLORS.warning, ref => `Deployment *skipped* for ${ref}`],
```

Comment:
So in our use case, "stopped" means that the run was cancelled by a user, while "skipped" means that another new run will take the lead?

Reply:
Yes exactly

Reply:
Another option instead of `skipped` is [superseded](https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_PipelineExecution.html#CodePipeline-Type-PipelineExecution-status:~:text=Superseded%3A%20While%20this%20pipeline%20execution%20was,and%20continued%20through%20the%20pipeline%20instead.) that is used by AWS codepipeline but I tought that `skipped` was more general and could be used in other context. 

What do you think ?

Reply:
Yes, I also prefer `skipped` 👍